catalog-spec/json-schema/catalog.json

```diff
@@ -42,6 +49,29 @@
           "type": "string",
           "const": "0.7.0"
         },
+        "stac_extensions": {
+          "title": "STAC extensions",
+          "anyOf": [
+            {
+              "title": "Reference to a JSON Schema",
+              "type": "string",
+              "format": "uri"
+            },
+            {
+              "title": "Reference to a core extension",
+              "type": "string",
+              "enum": [
+                "checksum",
+                "datacube",
+                "datetime-range",
+                "eo",
+                "pointcloud",
+                "sar",
+                "scientific"
```

Comment:
Should this list include API extensions? Or should we add them separately at /stac explicitly?

Reply:
I think that's separate issue regarding the API response, and since we had discussed recently about versioning the API separately, I think we nee to also talk about returning the API version as well.

Reply:
I'm agreeing with @matthewhanson here. API extensions could go in `/conformance` if we'd like to go the OGC way.